readthedocs/core/mixins.py

```diff
@@ -39,7 +41,11 @@ class HideProtectedLevelMixin:
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
 
+        privacy_level = list(PRIVACY_CHOICES)
+        privacy_level.remove((PROTECTED, _('Protected')))
+
         if self.instance is None or self.instance.privacy_level != PROTECTED:
-            privacy_level = list(PRIVACY_CHOICES)
-            privacy_level.remove((PROTECTED, _('Protected')))
             self.fields['privacy_level'].choices = privacy_level
+
+        if 'external_builds_privacy_level' in self.fields:
+            self.fields['external_builds_privacy_level'].choices = copy(privacy_level)
```

Comment:
We could convert `privacy_level` into a `tuple()` again in line 44 and avoid using `copy`.

Reply:
Yea, I'd prefer to avoid `copy` if possible. Is it needed for some reason? If so, we should comment why.